pkg/acsengine/const.go

```diff
@@ -40,6 +40,8 @@ const (
 	DefaultNetworkPolicy = "none"
 	// DefaultKubernetesNodeStatusUpdateFrequency is 10s, see --node-status-update-frequency at https://kubernetes.io/docs/admin/kubelet/
 	DefaultKubernetesNodeStatusUpdateFrequency = "10s"
+	// DefaultKubernetesHardEvictionThreshold is "memory.available<10%, see --eviction-hard at https://kubernetes.io/docs/admin/kubelet/
+	DefaultKubernetesHardEvictionThreshold = "memory.available<10%"
```

Comment:
According to the k8s docs, this default string should be:

`"memory.available<100Mi,nodefs.available<10%,nodefs.inodesFree<5%"`

Reply:
@jackfrancis thanks! I have updated the default string. We were proposing memory.available<10% as a default here, but now I think its better raised in the upstream k8s repo